test/fail_compilation/fail19729.d

```diff
@@ -0,0 +1,28 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19729.d(27): Error: `fail19729.C.__ctor` called with argument types `(string)` matches both:
+fail_compilation/fail19729.d(14):     `fail19729.C.Templ!string.this(string t)`
+and:
+fail_compilation/fail19729.d(14):     `fail19729.C.Templ!string.this(string t)`
+---
+*/
+module fail19729;
+
+mixin template Templ(T)
+{
+    this(T t)
+    {
+    }
```

Comment:
FYI: there's no requirement to format the tests and even in general `this(T t) {}` would be allowed ;-)